package com.wheredidikeepit.app

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import com.wheredidikeepit.app.ui.navigation.AppNavigation
import com.wheredidikeepit.app.ui.theme.WhereDidIKeepItTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContent {
            WhereDidIKeepItTheme {
                AppNavigation()
            }
        }
    }
}
